packages/style-engine/docs/using-the-style-engine-with-block-supports.md

```diff
@@ -0,0 +1,151 @@
+# Using the Style Engine to generate block supports styles
```

Comment:
Is the intention that this document is to be available in the developer handbook? If so, I _think_ we might need to add an entry in the `docs/manifest.json` file for it. Here's the line for the parent style engine readme: https://github.com/WordPress/gutenberg/blob/trunk/docs/manifest.json#L1815

From the looks of it, we can probably create a child page for this document to live in by copying one of the entries from the components package, e.g. [here's the entry for Notice](https://github.com/WordPress/gutenberg/blob/3b4526a7a91b6ec38d6bb067db650679ec9e3573/docs/manifest.json#L1053)

I imagine it's a matter of setting a title and a slug, the correct `markdown_source` and then setting `parent` to `"packages-style-engine"`?

Reply:
Thanks @andrewserong! Yes, I think it should be part of the docs. On it...